### ISSUE TITLE
linter always shows INFO[0000] Done without errors    

### ISSUE THREAD

**Issue by azhurbilo** [leominov/prometheus-devops-linter#3]
get rule example from test_data and remove required "env" label

```
---
groups:
  - name: HAProxy
    rules:
      - record: job:http_inprogress_requests:sum
        expr: sum(http_inprogress_requests) by (job)
      - alert: HAProxyDown
        expr: haproxy_up{env = "prod"} == 0
        for: 3m
        labels:
          severity: critical
          service: "{{ $labels.service }}"
          group: "{{ $labels.group }}"
        annotations:
          summary: HAProxy {{ $labels.group }} {{ $labels.instance }} недоступен
          description: HAProxy на сервере {{ $labels.instance }} группы {{ $labels.group }} не отвечает более 3 минут
          grafana_url: http://grafana.local/dashboard/db/haproxy-servers?from=now-30m&var-node={{ $labels.instance }}
          brief_summary: "HAProxyDown: {{ $labels.instance }}"
```

```
prometheus-devops-linter rules /data/common.yaml 
INFO[0000] Starting prometheus-devops-linter (version=1.5.0, branch=master, revision=8f3035aa5e240a1ffa7a7b134621cba23ba5e47d)... 
INFO[0000] Configuration path: /.prometheus-linter.yaml 
INFO[0000] Processing '/data/common.yaml' rules file... 
INFO[0000] Done without errors      
```

Actual result: Done without errors      

Expected result: Lint Warning -> "env" label is required

Also `export LOG_LEVEL=debug`
doesn't show any additional logs

**Comment by leominov** [comment]
Fixed in 1.5.1, thanks.

**Comment by azhurbilo** [comment]
cool, it works 👍 
thank you for fast fix
